lib/gringotts/gateways/monei.ex

```diff
@@ -107,7 +106,7 @@ defmodule Gringotts.Gateways.Monei do
   aliases to it (to save some time):
   ```
   iex> alias Gringotts.{Response, CreditCard, Gateways.Monei}
-  iex> opts = [currency: "EUR"] # The default currency is EUR, and this is just for an example.
+  iex> amount = %{amount: 40, currency: "EUR"}
```

Comment:
Doing this is okay, for testing purpose developer can use a `Map` but in case of code follow my comments in this PR it would awesome if we can somehow make it explicit that we support 2-3 money libs out there, maybe a bunch of grouped test cases just thinking?

Thoughts? @oyeb @kartikjagdale @arjun289 @SagarKarwande @zeus999

Reply:
I'd love to do that! And I'd love to add integration tests that make sure `ex_money` and `monetized` are compliant.
All that's stopping me is that I'll have to add them as an `optional` or `only: :test` dependency.

Reply:
And if we are adding them as dependency, I suggest we get them as `optional: :true` instead of `only: :test` as that will last us explicitly implement the protocol for them (which is currently happening via the fall back to `any`)

Reply:
@oyeb it will awesome if we have integration test for ex_money and monetized scenarios.

Reply:
@oyeb Yes, let's add them as `optional` and implement the protocol, make sure we have a fallback in any case.